include/benchmark/benchmark_api.h

```diff
@@ -154,6 +154,7 @@ BENCHMARK(BM_test)->Unit(benchmark::kMillisecond);
 #include <stdint.h>
 
 #include "macros.h"
+#include "benchmark_mpi.h"
```

Comment:
This include is making `benchmark_mpi.h` part of the benchmark API.  That's a bad idea, it's leaking implementation details.  You should call `initialize_mpi` at the beginning of `Initialize` and `finalize_mpi` at the end of `RunSpecifiedBenchmarks`.